CHANGELOG.md

```diff
@@ -67,7 +67,30 @@
   details.
 </details>
 
-## v1.32.0 - TBD
+## v1.33.0 - TBD
+
+## v1.32.0 - 2021-10
+
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+* docs(bigtable): add documentation for no channel refreshes ([#7373](https://github.com/googleapis/google-cloud-cpp/pull/7373))
+
+### [IAM](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/iam/README.md)
+
+* doc(iam): use SetIamPolicy() read-modify-write loop in sample ([#7288](https://github.com/googleapis/google-cloud-cpp/pull/7288))
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat(pubsub): configure SubscriberConnection with Options ([#7342](https://github.com/googleapis/google-cloud-cpp/pull/7342))
+* feat(pubsub): configure PublisherConnection with Options ([#7341](https://github.com/googleapis/google-cloud-cpp/pull/7341))
+* feat(pubsub): configure SubscriptionAdminConnection with Options ([#7339](https://github.com/googleapis/google-cloud-cpp/pull/7339))
+* feat(pubsub): configure SchemaAdminConnection with Options ([#7338](https://github.com/googleapis/google-cloud-cpp/pull/7338))
+* feat(pubsub): configure TopicAdminConnection with Options ([#7336](https://github.com/googleapis/google-cloud-cpp/pull/7336))
+* feat(pubsub): add policy options; default common options ([#7335](https://github.com/googleapis/google-cloud-cpp/pull/7335))
+* feat(pubsub): introduce SubscriberOptionList ([#7319](https://github.com/googleapis/google-cloud-cpp/pull/7319))
+* fix(pubsub): avoid deadlocks in publish flow control ([#7313](https://github.com/googleapis/google-cloud-cpp/pull/7313))
+* feat(pubsub): introduce PublisherOptionList (g::c::Options) ([#7308](https://github.com/googleapis/google-cloud-cpp/pull/7308))
```

Comment:
I think this can be included in the summary above. @dbolduc thoughts?

Reply:
yes, it would be captured by the above suggestion

Reply:
Done.